Makefile

```diff
@@ -236,7 +236,7 @@ tests: test
 TEST_FLAGS_LIST = ""\
 -O \
 --lcalc \
---lcalc,--closure-conversion,-O \
+--lcalc,-O \
```

Comment:
Erf, indeed, externals won't cope well with changing the call convention of functions. That test was useful though, I'll see if we can find another solution not disable it entirely.

(otherwise, there's still hope that closure conversion bugs will be caught by the C backend, but:
   - that includes a smaller number of tests
   - it's much less helpful for debugging them)

Reply:
Actually I think externals might work (they are not expected to be closures, just functions, closure conversion is done later on) ; but it's the basic operators `fold` or `map` that don't have the right type because they should accept closures as arguments.